### ISSUE TITLE
Fails with Error: Unknown @page '_nav' in toTree()

### ISSUE THREAD

**Issue by jantimon** [palantir/documentalist#71]
```
npx documentalist "src/**/*"
```

node version: 6

```
Error: Unknown @page '_nav' in toTree()
    at PageMap.toTree (_npx/61470/lib/node_modules/documentalist/dist/page.js:55:19)
    at MarkdownPlugin.compile (_npx/61470/lib/node_modules/documentalist/dist/plugins/markdown.js:42:27)
    at _npx/61470/lib/node_modules/documentalist/dist/documentalist.js:127:69
    at step (_npx/61470/lib/node_modules/documentalist/dist/documentalist.js:38:23)
    at Object.next (_npx/61470/lib/node_modules/documentalist/dist/documentalist.js:19:53)
    at step (_npx/61470/lib/node_modules/documentalist/dist/documentalist.js:23:97)
    at Object.next (_npx/61470/lib/node_modules/documentalist/dist/documentalist.js:19:53)
    at _npx/61470/lib/node_modules/documentalist/dist/documentalist.js:13:71
    at __awaiter (_npx/61470/lib/node_modules/documentalist/dist/documentalist.js:9:12)
    at Documentalist.documentFiles (_npx/61470/lib/node_modules/documentalist/dist/documentalist.js:114:16)
    at Documentalist.<anonymous> (_npx/61470/lib/node_modules/documentalist/dist/documentalist.js:102:44)
    at step (_npx/61470/lib/node_modules/documentalist/dist/documentalist.js:38:23)
    at Object.next (_npx/61470/lib/node_modules/documentalist/dist/documentalist.js:19:53)
    at _npx/61470/lib/node_modules/documentalist/dist/documentalist.js:13:71
    at __awaiter (_npx/61470/lib/node_modules/documentalist/dist/documentalist.js:9:12)
    at Documentalist.documentGlobs (_npx/61470/lib/node_modules/documentalist/dist/documentalist.js:98:16)

```

**Comment by giladgray** [comment]
@jantimon this is expected given the default options: https://palantir.github.io/documentalist/#IMarkdownPluginOptions.
name a page `_nav` or change that option.

**Comment by crazytoucan** [comment]
Maybe documentalist could improve its error messaging a bit in this case? This hit me too today, and it was not obvious how I was supposed to address the issue.

**Comment by giladgray** [comment]
fixed in af350da
